tests/importer/model_flattening.cpp

```diff
@@ -832,3 +830,132 @@ TEST(ModelFlattening, importedComponentsWithConnectionsToChildren)
     auto a = printer->printModel(modelUsingImports);
     EXPECT_EQ(e, a);
 }
+
+TEST(ModelFlattening, resolveFlattenCircularImportsComponents)
```

Comment:
This test really ought to be split in two . One for the resolving and another for the flattening.

Reply:
Same for the `resolveFlattenCircularImportsUnits` test.

Reply:
a107cfd4

Reply:
Can't be done wince flattening first requires resolving.